### DISCUSSION TITLE
Posso utilizar uma licença em outros computadores?

### DISCUSSION THREAD

**Discussion by ghost** [aclap-dev/video-downloadhelper#1370] [Help and Q&A]
### Your question

**Posso utilizar uma licença em outros computadores? na mesma conta do google ou a licença e local?**

### Addon & CoApp version and details

Quero sabe se tem como eu comprar a licença numa conta do google e utilizar a versão premium em outro computador logado com a conta que eu comprei a versão premium.

**Comment by jcfrog** [comment]
hi, you can use your license on all your devices. But it's not link to a Google account, you just have to validate your key on the computer you want: https://github.com/aclap-dev/video-downloadhelper/wiki/V9-interface#validate-your-license-key
